src/cli.rs

```diff
@@ -78,9 +79,15 @@ impl TrinConfig {
             )
             .arg(
                 Arg::with_name("external_addr")
-                .long("external-address")
-                .help("The public IP address and port under which this node is accessible")
-                .takes_value(true),
+                    .long("external-address")
+                    .help("The public IP address and port under which this node is accessible")
+                    .takes_value(true),
+            )
+            .arg(
+                Arg::with_name("private_key")
+                    .long("private-key")
+                    .help("Hex encoded 32 byte private key")
+                    .takes_value(true),
             )
```

Comment:
Private keys should not be taken as command line arguments, since they end up in terminal history and in the `/proc` file system.

Probably should pass a path, and read from there?

Reply:
Yup, this makes sense and I def agree. Any objections to me just creating an issue to track the update for now? And consider the update "critical" before any kind of initial release. This pr is mostly to help with simplicity of spinning up a testnet and all the various nodes - which becomes more complex if we require keyfiles. All of the private keys I'm using are not even real (`00.01`, `00.02`, etc).

Reply:
Maybe rename the argument to `unsafe_private_key`?

In any case, my review is nonbinding, I'm only a contributor :rofl:

Reply:
Hahaha #allcontributorsmatter. I updated the `long` param to `unsafe-private-key`. left the name `private_key` as is, since that's internal to the code. Tracking the update here: https://github.com/carver/trin/issues/36